lib/nerves_init_gadget/network_manager.ex

```diff
@@ -143,28 +143,31 @@ defmodule Nerves.InitGadget.NetworkManager do
       # make it discoverable
       %Mdns.Server.Service{
         domain: "_services._dns-sd._udp.local",
-        data: "#{opts.mdns_name}._services._dns-sd._udp.local",
+        data: "#{mdns_discovery_name(opts)}._services._dns-sd._udp.local",
```

Comment:
Should this have been kept `opts.mdns_name`? This makes a string like `"#{opts.mdns_name} (#{resolve_mdns_name(opts.mdns_domain)})._services._dns-sd._udp.local"` and that doesn't seem right.